### ISSUE TITLE
IPv6 failover

### ISSUE THREAD

**Issue by dgeo** [ovh/infrastructure-roadmap#114]
Allow using IPv6 as IPv4 failover.
For the moment one can only use server-assigned block.

**Comment by pinaraf** [comment]
I disagree, IPv6 failover implemented as IPv4 failover would suck, they would be utterly useless with any good IPv6 setup.
A great implementation (at least on the network side) of IPv6 is the one by online.net : each customer has a /48, and does as he wants with it.
This way, I could easily give a /56 for a server (per RFCs), and a /64 for OpenVPN. If my server dies and I need a new one, I configure the dhcp6 on the new server with the previous DUID and all is good.

**Comment by pinaraf** [comment]
=> https://github.com/ovh/infrastructure-roadmap/issues/117

**Comment by diamondzxd** [comment]
That actually makes more sense as an idea, but I'm curious about the implementation here.

**Comment by wzrdtales** [comment]
> I disagree, IPv6 failover implemented as IPv4 failover would suck, they would be utterly useless with any good IPv6 setup. A great implementation (at least on the network side) of IPv6 is the one by online.net : each customer has a /48, and does as he wants with it. This way, I could easily give a /56 for a server (per RFCs), and a /64 for OpenVPN. If my server dies and I need a new one, I configure the dhcp6 on the new server with the previous DUID and all is good.

As stated in the other ticket: while ovh has to provide ipv6 failover for it to be even remotely useful, you undercomplicate the matter here. A big network has to segment IPs to be scalable, so just handing out a bigger prefix wouldn't solve the problem of failover for OVH, there is more logic involved.

**Comment by Wild1145** [comment]
Being able to move IPv6 Addresses / Subnets between bare metal machines currently blocks my ability to adopt IPv6 and has done since IPv6 was able to be used for my workloads. The sooner we have some means to make the IP Addresses transient the better the ability to meaningfully adopt it

**Comment by thias** [comment]
I can see a valid use case for "IPv6 failover", which would now be called "OVH Additional Prefix", of course :wink:
Any small customers with just a couple of bare metal servers would be able to make a /64 prefix "float" between them, providing some level of IPv6 redundancy, even if only using a statically configured single /128 address on both.

**Comment by jslocinski** [comment]
🚀 I'm pleased to announce early release of OVHcloud's Additional IPv6 offering a flexible way to manage next-gen IP addressing with vRack-enabled products! 👉 Register now and grab one of the first /56 blocks during Alpha product stage: https://labs.ovhcloud.com/en/additional-ipv6-with-vrack/

**Comment by jslocinski** [comment]
Additional IPv6 is now in General Availability.
